composer.json

```diff
@@ -22,14 +22,15 @@
         "symfony/http-foundation": "^5.4",
         "symfony/monolog-bridge": "^5.4",
         "vkartaviy/retry": "^0.2",
-        "keboola/object-encryptor": "^1.0",
+        "keboola/object-encryptor": "dev-PS-3644",
         "monolog/monolog": "^2.5",
         "mustache/mustache": "^2.13",
         "keboola/common-exceptions": "^1.0",
         "keboola/sandboxes-api-php-client": "^6.7",
         "php": ">=7.4",
         "ext-zip": "*",
-        "keboola/artifacts": "^0.0.7"
+        "keboola/artifacts": "^0.0.7",
+        "keboola/job-queue-internal-api-php-client": "dev-pepa_PS-3651_newEncryption as 10.11.0"
```

Comment:
To sem fakt jdeš dávat? to udělá krásnou skoro circular dependency

Reply:
Ani nevím na co to tu je

Reply:
dobra otazka.. to bude blbe zkopirovany z daemona, pujde to pryc

Reply:
aha, tak to neni omyl :slightly_frowning_face: https://keboola.slack.com/archives/CQB468K63/p1658398123717389

Reply:
vyhozeno